Assets/_Scripts/Enemy/Enemy.cs

```diff
@@ -58,6 +65,10 @@ public void Shoot()
         currentProjectile.speed = projectileSpeed;
 
         currentProjectile.offset = 0;
```

Comment:
This isn't being used and we don't want to aim the bullets at the player we can remove it.

Reply:
removed